src/safari-extension/popup/popup.js

```diff
@@ -38,7 +40,28 @@ function onPopover(event) {
 
 function onMessage(event) {
   if (event.name === 'tickets') {
-    render(event.message, { grab, openext });
+    const templates = {
+      commit: settings.commitMessageFormat,
+      branch: settings.branchNameFormat,
+      command: settings.commandFormat,
+    };
+
+    const fmt = format(templates);
+
+    const enhance = ticket => ({
+      fmt: {
+        branch: fmt.branch(ticket),
+        commit: fmt.commit(ticket),
+        command: fmt.command(ticket),
+      },
+      ...ticket,
+    });
```

Comment:
This should be factored out because it's the same for both Safari and WebExtension browsers.